QuizApp/Login/LoginViewController+Design.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+import SnapKit
+
+extension LoginViewController: ConstructViewsProtocol {
```

Comment:
You can put this in the main VC file and have a `MARK` before. Current styleguide is a bit outdated 😕 
This is also ok, but we're trying out this new approach to reduce the number of files.